connection_optimization/do_external_connect.js

```diff
@@ -16,14 +16,17 @@ import parseURLParams from '../react/features/base/config/parseURLParams';
  */
 
 if (typeof createConnectionExternally === 'function') {
-    // URL params have higher proirity than config params.
+    // URL params have higher priority than config params.
     let url
         = parseURLParams(window.location, true, 'hash')[
                 'config.externalConnectUrl']
             || config.externalConnectUrl;
+    const isRecorder
```

Comment:
is this new logic?

Reply:
Yes, cause jibri needs to disable externalConnectURL in order automatic authentication(using localstorage values) to work.